litmus-portal/frontend/src/utils/promUtils.ts

```diff
@@ -1,6 +1,46 @@
-import { ChaosResultNamesAndNamespacesMap } from '../models/dashboardsData';
+/* eslint-disable no-unused-expressions */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-loop-func */
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+import { GraphMetric } from 'litmus-ui';
```

Comment:
Remove the unnecessary `disable linting statements` and do appropriate changes

Reply:
these cannot be removed because of comments and error logging to console, also due to the use case for looping.

Reply:
I have already removed the unnecessary ones.

Reply:
the comments can be broken down into multiple lines. It will be easy to read for others also. Please remove `max-len`

Reply:
The comments mentioned here are for the developers, to implement the change in litmus-ui and then integrate here, i have tried to keep them short but they are exceeding by some 4 to 5 words each, these comments which are exceeding in length are temporary till the feature is complete and timestamp based chaos metric filtering is live on the graphs. You may remove after updating litmus-ui and the function as mentioned in the comments.

Reply:
Even the temporary comments should adhere to the set standards cc: @arkajyotiMukherjee